lib/github.js

```diff
@@ -39,6 +39,13 @@ export default {
   createStatus: async (owner, repo, head_sha, entries) => {
     const github = await getRestClient();
 
+    const downloadLinks = entries.map(entry => `- [${entry.artifactId}](${entry.url})`).join("\n")
+    const pluginManager = entries.map(entry => `${entry.artifactId}:incrementals;${entry.groupId};${entry.version}&#60;/version>&#xA;&#60;/dependency>`).join("&#xA;")
+
+    let completeSummary = "### Incrementals download links:\n" + downloadLinks
```

Comment:
do you have an example of how this looks?

Reply:
See new output at https://github.com/jenkins-infra/incrementals-publisher/pull/88#issuecomment-1937816007